packages/flutter/lib/src/cupertino/switch.dart

```diff
@@ -141,7 +151,9 @@ class CupertinoSwitch extends StatefulWidget {
     this.autofocus = false,
     this.dragStartBehavior = DragStartBehavior.start,
   })  : assert(activeThumbImage != null || onActiveThumbImageError == null),
-        assert(inactiveThumbImage != null || onInactiveThumbImageError == null);
+        assert(inactiveThumbImage != null || onInactiveThumbImageError == null),
```

Comment:
Should we also assert the user has not set both the old and the new properties together?